indra/llcorehttp/_httplibcurl.cpp

```diff
@@ -487,12 +487,22 @@ void HttpLibcurl::policyUpdated(unsigned int policy_class)
         policy.stallPolicy(policy_class, false);
         mDirtyPolicy[policy_class] = false;
 
-        if (options.mPipelining > 1)
+        if (options.mPipelining)
         {
             // We'll try to do pipelining on this multihandle
-            check_curl_multi_setopt(multi_handle,
-                                     CURLMOPT_PIPELINING,
-                                     1L);
+            LL_INFOS("HttpLibcurl") << "HTTP Pipelining Enable." << LL_ENDL;
+            if (options.mMultiplexing)
```

Comment:
The nesting conditionals are probably wrong, in practice.  If mMultiplexing is on but mPipelining is off, multiplexing ends up disallowed.  Probably not intended.  One of those areas we'll have to sort out.  If the 'mPipelining' concept is changed to concurrency, this is more nearly correct.